src/context/providers/ProductProvider.js

```diff
@@ -35,7 +36,7 @@ const ProductProvider = ({ children }) => {
     filterData
   )(state, state?.products || []);
 
-  console.log(filteredProducts, state.productsFilter);
+  // console.log(filteredProducts, state.productsFilter);
```

Comment:
Remove comments

Reply:
Thanks for pointing out :) I will remove it in next PR.